src/gui/tray/UnifiedSearchResultsListModel.cpp

```diff
@@ -0,0 +1,675 @@
+/*
+ * Copyright (C) by Oleksandr Zolotov <alex@nextcloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+#include "UnifiedSearchResultsListModel.h"
+
+#include "account.h"
+#include "accountstate.h"
+#include "guiutility.h"
+#include "folderman.h"
+#include "networkjobs.h"
+
+#include <algorithm>
+
+#include <QAbstractListModel>
+#include <QDesktopServices>
+
+namespace {
+static QString imagePlaceholderUrlForProviderId(const QString &providerId)
```

Comment:
static is not needed inside an anonymous namespace. It already has internal linkeage.